File: figforge-code-engine/app/supabase_client.py
# supabase_client.py
from supabase import create_client
from dotenv import load_dotenv
import os

# Load environment variables from .env file
load_dotenv()

# Get Supabase URL and key from environment variables
supabase_url = os.getenv("SUPABASE_URL")
supabase_key = os.getenv("SUPABASE_KEY")

# Initialize Supabase client
supabase = create_client(supabase_url, supabase_key)

File: figforge-figma-plugin/src/app/supabaseClient.ts
import { createClient } from '@supabase/supabase-js';

// Your Supabase URL and public API key
const supabaseUrl = 'https://kpclehqrncegbgduiysb.supabase.co';
const supabaseKey =
  'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6ImtwY2xlaHFybmNlZ2JnZHVpeXNiIiwicm9sZSI6ImFub24iLCJpYXQiOjE2OTk1MTIzMTYsImV4cCI6MjAxNTA4ODMxNn0.1EZExw5NZdMaWWk5TdBF-F7pWsOapuKxseVh96kXlBQ';

// Create a single supabase client for interacting with your database
export const supabase = createClient(supabaseUrl, supabaseKey);
